src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java

```diff
@@ -201,6 +201,20 @@ CaseDetails submitEventForCitizen(
             @RequestParam Map<String, String> searchCriteria
     );
 
+    /**
+     * Retrieve case by Case Id
+     */
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/cases/{cid}",
+            headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE
+    )
+    CaseDetails getCase(
```

Comment:
story does say to call it `read` ;)

Reply:
does what it says on the tin.

I personally think the api clients should  be split between types.
caseWorkerApi
citizenApi 
Makes it easy to see whats meant for what.
getCase really just does what it says on the tin.

Reply:
personally there shouldn't be 2 different api's are they are identical ;)